### DISCUSSION TITLE
MFA Biometric "Push Notification"

### DISCUSSION THREAD

**Discussion by AmeerErg** [neos-sdi/adfsmfa#238] [Q&A] [answered]
Hi,

Does MFA Biometric support "Push Notification"? 

Push Notification for example is to  confirm or approve the request using the phone.

Regards,

**Comment by redhook62** [comment, marked as answer]
Hi @AmeerErg 

To answer you directly, No.

The protocol is standardized (WebAuthN/FIDO2).

Enrollment ceremony may display notifications, but this depends on the device, to a lesser extent, on the WebAuthN security configuration. it's the authenticator of the device or the browser that displays messages, 
for example to request authorization to store the authenticator model (Finger Print, Yubikey, IOS, etc.), 

For the Authentication part, this can also happens, for example to ask the user for confirmation for the use of Biometrics.
each use case is possibly different depending on your device
